### PR TITLE
#160577471 Setup and deploy base application to Heroku

### PR DESCRIPTION
**Pull request by lukorito**
### **What does this PR do?**
This PR sets up and deploys the application on heroku to enable users to access it.
###  **Description of Task to be completed?**
- Enable users to access the live app using given urls
- When a PR is raised, a review app is created that mirrors the PR
- App is divided into `staging` for testing and `production` for the live app
### **How should this be manually tested?**
It can be tested by using the following links:
`http://ah-shakas-staging.herokuapp.com/` for stating
`http://ah-shakas.herokuapp.com/` for production
###  **What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/story/show/160577471
###  **Screenshots**
screenshot 2018-09-25 11 24 00

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
+django-heroku==0.3.1
+gunicorn==19.9.0
+psycopg2==2.7.5
+psycopg2-binary==2.7.5
 coverage==4.5.1
-Django==2.1.1
-django-cors-middleware==1.3.1
-django-extensions==2.1.2
-djangorestframework==3.8.2
-djangorestframework-jwt==1.11.0
```
Comment (FawazFarid) [inline requirements.txt]: I don't understand this change, these are important libraries and frameworks you've removed. Why?

File: .gitignore [hunk 2]
```diff
@@ -94,13 +94,3 @@ ENV/
...
-# vscode
-.vscode
-
-# vim temporary files
-.*.swp
-.*.swn
-.*.swo
-
-# vim python config
-.ycm_extra_conf.py
```
Comment (FawazFarid) [inline .gitignore]: We also want to ignore all this files you removed here

**Review by FawazFarid — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (lukorito) [reply requirements.txt]: There are available in the current requirements.txt

File: requirements.txt [hunk 3]
```diff
@@ -17,3 +19,7 @@ gunicorn==19.9.0
 psycopg2-binary==2.7.5
 whitenoise==4.1
 wrapt==1.10.11
+django-cors-middleware==1.3.1
+django-extensions==2.1.2
+djangorestframework==3.8.2
+djangorestframework-jwt==1.11.0
```
Comment (FawazFarid) [inline requirements.txt]: There are still duplicate libraries, please address this.

**Review by FawazFarid — CHANGES_REQUESTED** [review]

**Review by FawazFarid — APPROVED** [review]

**Pull request merged**